SecurityPkg/Include/Library/Tpm2CommandLib.h

```diff
@@ -381,12 +408,12 @@ Tpm2NvUndefineSpace (
 EFI_STATUS
 EFIAPI
 Tpm2NvRead (
-  IN      TPMI_RH_NV_AUTH    AuthHandle,
-  IN      TPMI_RH_NV_INDEX   NvIndex,
-  IN      TPMS_AUTH_COMMAND  *AuthSession  OPTIONAL,
-  IN      UINT16             Size,
-  IN      UINT16             Offset,
-  IN OUT  TPM2B_MAX_BUFFER   *OutData
+  IN      TPMI_RH_NV_AUTH AuthHandle,
+  IN      TPMI_RH_NV_INDEX NvIndex,
+  IN      TPMS_AUTH_COMMAND *AuthSession, OPTIONAL
+  IN      UINT16                    Size,
+  IN      UINT16                    Offset,
+  IN OUT  TPM2B_MAX_BUFFER          *OutData

```

Comment:
I'm guessing we can drop it and uncrustify won't complain

Reply:
Unfortunately, as funky it looks. This is what uncrustify wants..